### ISSUE TITLE
[BUG] 'Microsoft.NET.Sdk.iOS.Manifest-7.0.100 16.4.7125' has incompatible package type.

### ISSUE THREAD

**Issue by alaskanrogue** [dotnet/maui#18604]
### Description

Beginning this afternoon, the error message "Process cannot be executed on XMA server. Could not find Microsoft.iOS in /Users/marcgeorge/Library/Caches/Xamarin/XMA/SDKs/dotnet/packs/Microsoft.iOS.Sdk/16.4.7125" is received during build.

When attempting to install the package, which was published today, 11/7/2023, the package required by my application, the error message "NU1213: The package Microsoft.NET.Sdk.iOS.Manifest-7.0.100 16.4.7125 has a package type DotnetPlatform that is incompatible with this project." is received.

All packages in the project are up to date, however, include pre-releases.

Builds have been attempted on both Windows and Mac with the latest versions of VS Pr e with same results.

### Steps to Reproduce

Build a MAUI iOS project with the necessary packages from those noted below.

ios  17.0.8477-net8-rc2.2/8.0.100-rc.2
```
"CommunityToolkit.Maui" Version="6.1.0" />
"CommunityToolkit.Maui.Core" Version="6.1.0" />
"CommunityToolkit.Maui.MediaElement" Version="2.0.0" />
"Geolocator.Plugin" Version="1.0.2" />
"Microsoft.Azure.NotificationHubs" Version="4.1.0" />
"Microsoft.CognitiveServices.Speech" Version="1.33.0" />
"Microsoft.Data.Edm" Version="5.8.5" />
"Microsoft.Data.OData" Version="5.8.5" />
"Microsoft.Data.Services.Client" Version="5.8.5" />
"Microsoft.Extensions.ApiDescription.Client" Version="8.0.0-rc.2.23480.2">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
"Microsoft.Extensions.Features" Version="8.0.0-rc.2.23480.2" />
"Microsoft.Extensions.Primitives" Version="8.0.0-rc.2.23479.6" />
"Newtonsoft.Json" Version="13.0.3" />
"NSwag.ApiDescription.Client" Version="14.0.0-preview009">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
"Plugin.Maui.Audio" Version="2.0.0" />
"System.ServiceModel.Duplex" Version="6.0.0" />
"System.ServiceModel.Federation" Version="6.1.0" />
"System.ServiceModel.Http" Version="6.1.0" />
"System.ServiceModel.NetTcp" Version="6.1.0" />
"System.ServiceModel.Primitives" Version="6.1.0" />
"System.ServiceModel.Security" Version="6.0.0" />
"System.Spatial" Version="5.8.5" />
"Xamarin.Essentials" Version="1.8.0" />
```
### Link to public reproduction project repository

NA

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

7.0.96

### Affected platforms

iOS

### Affected platform versions

ios 16.4.8968-net8-rc2/8.0.100-rc.2

### Did you find any workaround?

No

**Comment by mattleibow** [comment]
I am wondering if this is because you are referencing `"Xamarin.Essentials" Version="1.8.0" />`. Xamarin.Essentials is only supported on classic Xamarin. .NET MAUI has a new version of Essentials that comes in the framework so no includes are necessary.

**Comment by alaskanrogue** [comment]
The project is a DotNet MAUI application. I will try to remove that NuGet again. 

Anyway, I was able to work around the issue. Other that upgrading VS this afternoon, I don't know what brought about the problem. Anyway,,,

1. Disconnect and removed the connect to the Mac.
2. Rebooted the PC.
3. Deleted the vs, bin and obj directories of the project.
4. Deleted the C:\Users\xxx\AppData\Local\Xamarin\iOS\Cache directory.
6. Rebooted the Mac.
7. Deleted the ~/Library/Caches/Xamarin/XMA/SDKs/dotnet directory on the Mac.
8. Re-paired the PC to the Mac.

The PC reloaded all necessary libraries during the pairing, including the missing NuGet. I was able to build the app.
